koku/masu/processor/orchestrator.py

```diff
@@ -260,20 +261,49 @@ def start_manifest_processing(  # noqa: C901
                 LOG.info(log_json(tracing_id, msg="download queued", schema=schema_name))
 
         manifest_list = [manifest.get("manifest_id") for manifest in manifest_list]
+        LOG.info(
+            log_json(
+                "start_manifest_processing",
+                msg="created manifest list",
+                report_tasks=report_tasks,
+                summarize_reports=self._summarize_reports,
+                schema=schema_name,
+            )
+        )
         if report_tasks:
             if self._summarize_reports:
                 reports_tasks_queued = True
                 hcs_task = collect_hcs_report_data_from_manifest.s().set(queue=HCS_Q)
```

Comment:
Should we just log this one too while we are at it?

Reply:
Sure thing! I was being a bit conservative since I wasn't sure about the `str` or `repr` of that object. But I can test it locally to see how it behaves.

Reply:
Is this what you meant?
```
{'message': 'created summary_task signature', 'tracing_id': 'start_manifest_processing', 'schema': 'org1234567', 'summary_task': 'masu.processor.tasks.summarize_reports(manifest_list=[20], ingress_report_uuid=None)', 'hcs_task': 'hcs.tasks.collect_hcs_report_data_from_manifest()'}
```

Reply:
Added.